lib/mongoid/criteria/scrollable/iterator.rb

```diff
@@ -8,6 +8,10 @@ def initialize(previous_cursor:, next_cursor:)
           @previous_cursor = previous_cursor
           @next_cursor = next_cursor
         end
+
+        def first_cursor
+          next_cursor.class.new(nil, next_cursor.sort_options)
```

Comment:
Add an attribute and memoize.

```suggestion
          @first_cursor ||= next_cursor.class.new(nil, next_cursor.sort_options)
```